ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.17.yaml

```diff
@@ -98,23 +104,28 @@ tests:
       cli: latest
       commands: |
         oc -n cert-manager-operator rollout status deployment cert-manager-operator-controller-manager
+      dependencies:
+      - env: OO_BUNDLE
+        name: cert-manager-operator-bundle
       from: src
       resources:
         requests:
           cpu: 100m
-    - ref: fips-check-fips-or-die
     - as: test
       cli: latest
       commands: |
+        #!/bin/bash
+        curl https://patch-diff.githubusercontent.com/raw/openshift/cert-manager-operator/pull/298.diff > pr.diff
+        git apply pr.diff
         E2E_GINKGO_LABEL_FILTER='"Platform: isSubsetOf {AWS} && !TechPreview"' make test-e2e
       from: src
       resources:
         requests:
           cpu: 100m
-    workflow: optional-operators-ci-operator-sdk-aws
+    workflow: optional-operators-ci-operator-sdk-generic-claim
 - as: e2e-operator-tech-preview
   optional: true
-  run_if_changed: ^(api/.*|pkg/operator/setup_manager\.go|pkg/operator/starter\.go|pkg/controller/.*|bindata/istio-csr|test/e2e/.*|Makefile)$
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
nit: Wouldn't run_if_changed be more apt by adding just api, config, images, pkg, test, go.mod? Yeah, the drawback would be updating this list when something new gets added to repo.

Reply:
The idea is to run this job more frequently as it would run all platform agnostic cases. Although it's named as `-tech-preview` and was added because of separating tech-preview feature test results into a separate job (regular one is served as `blocking`, while this is served as `informing`), increase its frequency would help improve our presubmit CI coverage.

https://github.com/openshift/release/blob/dcfb00f08941a46d36915f2d82485a003bb82bf0/ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.17.yaml#L127-L135